app/assets/stylesheets/modules/home.scss

```diff
@@ -0,0 +1,47 @@
+$good_dog_font: "GoodDog";
+
+.body-home-page {
+  background: url("dash.jpg") no-repeat center center fixed;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+}
+
+.new-bar {
+  padding: 0;
+}
+
+.bar {
+  max-width: 600px;
+}
+
+.search-button {
+  padding-left: 0;
+}
+
+.home-title {
+  -webkit-animation-duration: 1.7s;
+  -webkit-animation-delay: 2s;
+  -webkit-animation-iteration-count: 1;
+  position: absolute;
+  top: 30%;
+  left: 25%
+}
+
+.new_title {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.